File: IntroPC/Tarea 6/Primo/primo.py
sw=0
num=input("ingrese un numero: ")
i=2
while i<=(num-1):
    comp=num%i
    if comp==0:
        print"el numero no es primo\n"
        sw=1
        break
        
    i=i+1
if sw==0:
    print"el numero es primo\n"
    

File: IntroPC/Tarea 6/Primo/primo.cpp
#include <cstdio>
#include <cmath>

using namespace std;

int main (void)
{
  int num,i,comp,sw;
  printf("ingrese un numero: ");
  scanf("%d",&num);
  for(i=2;i<=(num-1);i++)
    {
      comp=num%i;
      if (comp==0)
	{
	  printf("el numero no es primo\n");
	  sw=1;
	  break;
	  }
    }
  if (sw==0)
    {
        printf("el numero es primo\n");
    }
  return 0;

}
